cmd/juju/machine/add.go

```diff
@@ -120,18 +167,18 @@ type addCommand struct {
 func (c *addCommand) Info() *cmd.Info {
 	return jujucmd.Info(&cmd.Info{
 		Name:    "add-machine",
-		Args:    "[<container>:machine | <container> | ssh:[user@]host | winrm:[user@]host | placement]",
-		Purpose: "Start a new, empty machine and optionally a container, or add a container to a machine.",
+		Args:    "[<container-type>[:<machine-id>] | (ssh|winrm):[<user>@]<host> | <placement>]",
+		Purpose: "Provision a new machine or assign one to the model",
```

Comment:
Just need a full stop at the end of this string.

Reply:
Fixed :)